example/main.go

```diff
@@ -36,6 +36,11 @@ func Helper(log logr.Logger, msg string) {
 
 func helper2(log logr.Logger, msg string) {
 	log.WithCallDepth(2).Info(msg)
+
+	// Underlier is currently unusable (https://github.com/go-logr/logr/issues/63).
```

Comment:
We don't need this comment in this PR, do we?

Reply:
It explains why code coverage isn't at 100%, but I can also take it out.

Reply:
It's unrelated to the rest of this.